mlrun/projects/project.py

```diff
@@ -2002,6 +2002,25 @@ def save_workflow(self, name, target, artifact_path=None, ttl=None):
         workflow_engine = get_workflow_engine(workflow_spec.engine)
         workflow_engine.save(self, workflow_spec, target, artifact_path=artifact_path)
 
+    def get_run_status(
+        self,
+        run,
+        timeout=None,
+        expected_statuses=None,
+        notifiers: RunNotifications = None,
+    ):
+        warnings.warn(
+            "This method is deprecated. Use `timeout` parameter in `project.run()` method instead",
+            PendingDeprecationWarning,
+        )
```

Comment:
```suggestion
        warnings.warn(
            "This will be deprecated in 1.4.0, and will be removed in 1.6.0"
            "Use `timeout` parameter in `project.run()` method instead",
            PendingDeprecationWarning,
        )
```